pkg/cmd/cmd_run.go

```diff
@@ -10,30 +10,31 @@ import (
 
 	"github.com/urfave/cli/v2"
 	"kusionstack.io/kpm/pkg/errors"
+	"kusionstack.io/kpm/pkg/oci"
 	"kusionstack.io/kpm/pkg/opt"
 	pkg "kusionstack.io/kpm/pkg/package"
 	"kusionstack.io/kpm/pkg/reporter"
 	"kusionstack.io/kpm/pkg/runner"
+	"kusionstack.io/kpm/pkg/settings"
 	"kusionstack.io/kpm/pkg/utils"
 )
 
 // NewRunCmd new a Command for `kpm run`.
-func NewRunCmd() *cli.Command {
+func NewRunCmd(settings *settings.Settings) *cli.Command {
 	return &cli.Command{
 		Hidden: false,
 		Name:   "run",
 		Usage:  "compile kcl package.",
 		Flags: []cli.Flag{
-			// The kcl package tar executed.
-			&cli.StringFlag{
-				Name:  "tar",
-				Usage: "The kcl package tar will be executed",
-			},
 			// The entry kcl file.
 			&cli.StringFlag{
 				Name:  "input",
 				Usage: "a kcl file as the compile entry file",
 			},
+			&cli.StringFlag{
+				Name:  "tag",
```

Comment:
Why called `tag` instead of `oci`.

Reply:
The `tag` is not used to instead `oci`, I have changed the `kpm run -- oci` mode to `kpm run <oci_url>`, `tag` is used to describe the version in `kpm run --tag v0.0.1 <oci_url>`